app/containers/ImageSkeletonPlaceholder/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import { StyleSheet, View } from 'react-native';
+import FastImage, { FastImageProps } from 'react-native-fast-image';
+import SkeletonPlaceholder from 'react-native-skeleton-placeholder';
+
+const ImageSkeletonPlaceholder = ({ style, ...otherProps }: FastImageProps): React.ReactElement => {
```

Comment:
This naming implies it's just a placeholder, when it's really both image and placeholder/skeleton.
Can you rename it to `ImageWithSkeleton`?
This way is easier to read the code.

Reply:
Good point, fixed ✅